src/app/components/home/home.component.html

```diff
@@ -1,89 +1,126 @@
-<div id="hero-image" class="h-100">
-  <!-- Title -->
-  <div class="container">
-    <h1 id="website-title" class="text-break">Ecosounds</h1>
+<!-- Hero image -->
+<section id="hero">
+  <div>
+    <h1 class="invisible">ECOSOUNDS</h1>
+    <!-- Scalable title, will resize to any display size -->
+    <svg width="80%" viewBox="-3 1 108 17">
+      <title>ECOSOUNDS</title>
+      <text x="0" y="15">ECOSOUNDS</text>
+    </svg>
   </div>
 
-  <!-- CMS -->
-  <div class="container-fluid" style="padding-bottom: 3rem">
-    <section class="container">
-      <baw-cms [page]="page"></baw-cms>
-    </section>
+  <caption>
+    Photo: QUT Ecoacoustics Staff
+  </caption>
+</section>
+
+<!-- Who are we? -->
+<section id="about_us">
+  <div class="container">
+    <h2>Who Are We?</h2>
+    <p>
+      Welcome! Ecosounds is a repository of environmental audio recordings. This
+      website facilitates the management, access, visualization, and analysis of
+      environmental acoustic data. It uses the Acoustic Workbench software which
+      is is open source and available from GitHub.
+    </p>
+    <p>
+      The website is run by the QUT Ecoacoustics Research Group to support
+      bioacoustics and ecoacoustics research.
+    </p>
   </div>
+</section>
 
-  <!-- Description of our process -->
-  <div class="container-fluid" style="padding-bottom: 3rem">
-    <section class="container">
-      <h2>Our Process</h2>
+<!-- What we do -->
+<section id="what_we_do">
+  <div class="container">
+    <h2>What We Do</h2>
 
-      <div class="justify-content-center row">
-        <div class="col-lg-4 col-sm-6 mb-4">
-          <div class="card h-100">
-            <h4 class="card-header text-center">Environment</h4>
-            <div class="card-body">
-              <p class="card-text" ng-reflect-ng-class="[object Object]">
-                Faunal vocalisations and other human-audible environmental
-                sounds
-              </p>
-            </div>
-          </div>
-        </div>
+    <div class="card-deck justify-content-center">
```

Comment:
`card-deck` class has been dropped in bootstrap 5. This will need to be re-implemented

Reply:
> Dropped .card-deck in favor of our grid. Wrap your cards in column classes and add a parent .row-cols-* container to recreate card decks (but with more control over responsive alignment).